CONTRIBUTING.md

```diff
@@ -18,6 +18,7 @@ This documented is primarily aimed at the following repositories:
 - [storage](https://github.com/containers/storage)
 - [libhvee](https://github.com/containers/libhvee)
 - [psgo](https://github.com/containers/psgo)
+- [ramalama](https://github.com/containers/ramalama)
```

Comment:
Why is this added here? The reason I only listed these repos here are because they are go repos controlled by the podman team and this document contains golang specific information around how we do dependency updates there.
As such I don't think adding this here is appropriate. Looks like you already have all the importan info copied anyway: https://github.com/containers/ramalama/blob/main/CONTRIBUTING.md so I don't see what this is adding.